packages/react-components/src/templates/AnalyticsPageHeader.tsx

```diff
@@ -28,19 +33,22 @@ const AnalyticsPageHeader: React.FC = () => (
         </Paragraph>
       </div>
       <TabNav>
-        {isEnabled('DISPLAY_ANALYTICS_PRODUCTIVITY') && (
-          <TabLink
-            href={analytics({}).productivity({}).$}
-            Icon={ProductivityIcon}
-          >
-            Resource & Data Sharing
-          </TabLink>
-        )}
-        {isEnabled('DISPLAY_ANALYTICS_COLLABORATION') && (
+        <TabLink
+          href={analytics({}).productivity({}).$}
+          Icon={ProductivityIcon}
+        >
+          Resource & Data Sharing
+        </TabLink>
+        {isEnabled('DISPLAY_ANALYTICS_BETA') && (
           <TabLink href={analytics({}).collaboration({}).$} Icon={TeamIcon}>
             Collaboration
           </TabLink>
         )}
+        {isEnabled('DISPLAY_ANALYTICS_BETA') && (
```

Comment:
It seems that on Figma `Engagement` is the last tab
![Screenshot 2024-07-08 at 13 42 39](https://github.com/yldio/asap-hub/assets/16595804/717b4c49-4bb7-479c-90d4-2b22f222a96c)